torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2346,8 +2346,8 @@ Tensor symeig_backward(const std::vector<torch::autograd::Variable> &grads, cons
   // leads to stable gradient updates, and retains symmetry of the updated matrix if it
   // were updated by a gradient based algorithm.
   TORCH_CHECK(eigenvectors,
-           "symeig_backward: Setting eigenvectors to false in torch.symeig doesn't compute eigenvectors ",
-           "and hence we cannot compute backward. Please use torch.symeig(eigenvectors=True)");
+           "symeig_backward: torch.symeig(A, eigenvectors=False) is not differentiable. ",
+           "Please use torch.linalg.eigvalsh(A)");
```

Comment:
"Use torch.linalg.eigvalsh(A), instead"

Reply:
Living the last few years in the UK has gotten the best of me.

Reply:
There are a number of constructions similar to this one in other errors. As it's not critical, I will not touch the others, as doing so would conflict with other PRs that have already been imported to phabricator.